source/nanoFramework.Json/JsonConvert.cs

```diff
@@ -397,15 +464,15 @@ private static JsonToken Deserialize()
 				throw new Exception("unexpected lexical token during json parse");
 			}
 			return result;
-		}
+		}	// end of Deserialize()
```

Comment:
No need for comments like this, as VS can easily shrink functions and this does not conform to the C# nF code style

Reply:
OK